### PR TITLE
Unify docker compose

### PR DESCRIPTION
**Pull request by IuryNogueira**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,12 @@
 # espoo-dev
 
+Run project with docker  🐳
 
-# doc
+```bash
+docker-compose -f docker-compose.dev.yml up
```
Comment (edimossilva) [inline README.md:6]: I think that is simpler if we use docker-compose.yml for dev and docker-compose.prod.yml for prod.
During daily usage have to add `-f docker-compose.dev.yml` to every command is not so coll, what do you think?

File: README.md [hunk 2]
```diff
@@ -1,7 +1,12 @@
...
+```
```
Comment (edimossilva) [inline README.md:7]: Also add the urls to access the apps

**Review by edimossilva — CHANGES_REQUESTED** [review]
link the issue (next time)
I am checking why CI is not running for your PR

**Comment by edimossilva** [comment]
@IuryNogueira Very nice, the readme is so pretty

**Comment by IuryNogueira** [comment]
> @IuryNogueira Very nice, the readme is so pretty

Thanks!

**Pull request merged**